core/server/apps/private-blogging/index.js

```diff
@@ -1,14 +1,13 @@
-var urlService = require('../../services/url'),
+const urlService = require('../../services/url'),
     common = require('../../lib/common'),
     middleware = require('./lib/middleware'),
     router = require('./lib/router'),
     registerHelpers = require('./lib/helpers'),
     // routeKeywords.private: 'private'
-    PRIVATE_KEYWORD = 'private',
-    checkSubdir;
+    PRIVATE_KEYWORD = 'private';
 
-checkSubdir = function checkSubdir() {
-    var paths;
+let checkSubdir = function checkSubdir() {
```

Comment:
Do we need to redundantly define the function here? More of a question for @kirrg001, but just thinking out loud :smile

Reply:
You could ask this question earlier, I don't invent it in this PR :)

Reply:
Definitely redundant, good catch 👍 

@tiendq No need to remove the function, the PR is only about ES6 :)